draft-jimenez-t2trg-mud-coap.md

```diff
@@ -208,20 +267,20 @@ REQ: GET coap://rd.company.com/rd-lookup/res?rt=mud
 RES: 2.05 Content
      Content-Format: TBD123456 (application/coral+cbor@identity)
 
-     rd-item <coap://[2001:db8:3::101]/mud/box> { rt "mud" }
-     rd-item <coap://[2001:db8:3::102]/mud/switch> { rt "mud" }
-     rd-item <coap://[2001:db8:3::102]/mud/lock> { rt "mud" }
-     rd-item <coap://[2001:db8:3::103]/mud/light> { rt "mud" }
+     rd-item <coap://[2001:db8:3::101]/mud/box> { rel "mud-file" }
+     rd-item <coap://[2001:db8:3::102]/mud/switch> { rel "mud-file" }
+     rd-item <coap://[2001:db8:3::102]/mud/lock> { rel "mud-file" }
+     rd-item <coap://[2001:db8:3::103]/mud/light> { rel "mud-file" }
 ~~~
 
 ### Multicast
```

Comment:
Possible additions:
- "All MUD nodes" multicast address
- "All MUD controller" multicast address

Reply:
- Requirement: packets need to be allowed to get through
- Scopes of addresses

Reply:
Well-known endpoint for controllers

Reply:
Additional methods for finding the interface(s) of MUD controllers.

Reply:
Restructure document: Well-known resources first, then methods for discovery.